rockchip_linux_defconfig: enable emc2305 sensors

nascs opened the pull request:
Signed-off-by: Nascs Fang <nascs@radxa.com>

RadxaYuntian commented on arch/arm64/configs/rockchip_linux_defconfig:
1. 文件结尾要有空格
2. 建议是用make menuconfig先开启需要选项后，用make savedefconfig重新生成更新后的defconfig，然后再替换对应文件

nascs replied:
我生成的 defconfig 和 rockchip_linux_defconfig 差异会有点大

RadxaYuntian replied:
那要确保你的内核不是通过我们bsp进行配置的，不然会添加一些额外的配置

nascs committed:
rockchip_linux_defconfig: enable emc2305 sensors

Signed-off-by: Nascs Fang <nascs@radxa.com>

nascs committed:
arm: dts: radxa cm3 rpi cm4 io: fix emc2301 registration failure problem

The error log:
  [    9.403071] thermal_sys: failed to build thermal zone soc-thermal: -2
  [   12.657120] rockchip-thermal fe710000.tsadc: failed to register sensor 0: -517
  [   12.659606] rockchip-thermal fe710000.tsadc: failed to register sensor[0] : error = -517

Signed-off-by: Nascs Fang <nascs@radxa.com>

nascs commented:
CONFIG_DMABUF_DEBUG=y

这个内核配置里面没有
